### PR TITLE
Add kintasrc to git during init

### PR DESCRIPTION
**Pull request by michgauz**

### CODE REVIEW

File: kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt [hunk 1]
```diff
@@ -24,8 +25,12 @@ object Init : CliktCommand(name = "init", help = "Initializes kinta in a project
         copyResource("com.dailymotion.kinta.Workflows", "kintaSrc/src/main/resources/META-INF/services/com.dailymotion.kinta.Workflows")
         copyResource("build.gradle.kts", "kintaSrc/build.gradle.kts")
         copyResource("gitignore", "kintaSrc/.gitignore")
+        copyResource("kinta.properties", "kintaSrc/kinta.properties")
 
+        //Assemble kintaSrc
         Gradle(File("kintaSrc")).executeTask("assemble")
```
Comment (martinbonnin) [inline kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: :grin:

File: kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt [hunk 2]
```diff
@@ -24,8 +25,12 @@ object Init : CliktCommand(name = "init", help = "Initializes kinta in a project
...
+        //Add kintaSrc to Git
+        GitIntegration.add(File("kintaSrc").path)
```
Comment (martinbonnin) [inline kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: Same here.

Comment (martinbonnin) [inline kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: Maybe write a message that `kintaSrc` has been added ? 

"This project is now setup to use kinta. You can define your workflows in kintaSrc/src/main/.... Take a look around. If you like what you see, commit your kintaSrc directory. It is meant to be in source control."

File: kinta-cli/src/main/resources/build.gradle.kts [hunk 3]
```diff
@@ -18,10 +18,10 @@ configure<JavaPluginExtension> {
 }
 
 dependencies {
-    api("com.dailymotion.kinta:kinta-lib:0.1.0")
+    api("com.dailymotion.kinta:kinta-lib:0.1.1")
     // The builtin workflows are in [com.dailymotion.kinta.BuiltinWorkflows].
     // Comment this line if you don't want the builtin kinta workflows
-    implementation("com.dailymotion.kinta:kinta-workflows:0.1.0")
+    implementation("com.dailymotion.kinta:kinta-workflows:0.1.1")
```
Comment (martinbonnin) [inline kinta-cli/src/main/resources/build.gradle.kts]: :+1: 

We could also replace that at runtime so we're sure we always bundle the latest version

File: kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt (hunk 1 above)
Comment (michgauz) [reply kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: 👍

File: kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt (hunk 2 above)
Comment (michgauz) [reply kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: 👍

Comment (michgauz) [reply kinta-cli/src/main/kotlin/com/dailymotion/kinta/command/Init.kt]: done

File: kinta-cli/src/main/resources/build.gradle.kts (hunk 3 above)
Comment (michgauz) [reply kinta-cli/src/main/resources/build.gradle.kts]: Done

**Pull request merged**